modules/virt-virtctl-commands.adoc

```diff
@@ -52,8 +52,17 @@ the specified port of the node.
 |`virtctl console <vmi_name>`
 |Connect to a serial console of a virtual machine instance.
 
-|`virtctl vnc <vmi_name>`
-|Open a VNC connection to a virtual machine instance.
+|`virtctl vnc --kubeconfig=$KUBECONFIG <vmi_name>`
+|Open a VNC connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a Virtual Network Client (VNC) which requires a remote viewer on your local machine.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only <vmi-name>`
+|Display the port number and connect manually to the virtual machine instance by using any viewer through the VNC connection.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi-name>`
```

Comment:
This and `virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only <vmi-name>` works the same way
No need for a separate line

Reply:
Change was made. Removed the additional command syntax.